style.css

```diff
@@ -1,5 +1,11 @@
 /* Typography */
 
+
+body {
```

Comment:
This is a smaller issue, but it's a best practice to style elements in the order that they appear in a HTML document, if there is one. In this case, `body` always comes after `html` tags, so this CSS block should come after the `html` one.